python/pp_ser.py

```diff
@@ -709,12 +689,40 @@ def __re_def(self):
             self.__check_intent_in(splitted[1])
         return m
 
+    # Produce the USE statement and append it to l
+    def __produce_use_stmt(self):
+        if (self.__use_stmt_in_module == True):
+            return
+        calls_pp = [c for c in self.__calls if     c.startswith('ppser')]
+        calls_fs = [c for c in self.__calls if not c.startswith('ppser')]
+        ncalls = len(calls_pp) + len(calls_fs)
+        if ncalls > 0:
+            calls_pp += ['ppser_savepoint', 'ppser_serializer', 'ppser_serializer_ref',
+                         'ppser_intlength', 'ppser_reallength', 'ppser_realtype', 'ppser_zrperturb']
```

Comment:
Yeah it's the only place. I can move it before the `if` or as a constant in the pp_ser